frontend/i18n.config.ts

```diff
@@ -5,10 +5,10 @@ export default defineI18nConfig(() => ({
   missing: (locale, key, vm) => {
     // TODO(grady) figure out how to skip this if we're in production + just log.
     // Consider using process.env.NODE_ENV == 'prod', etc.
-    const fn = inject('handleMissingTranslation')
+    const fn = vm?.appContext.app.$nuxt.$missingTranslations.handleMissingTranslation
     if (fn) {
-      const callable = fn as (locale: string, key: string) => void
-      callable(locale, key)
+      // const callable = fn as (locale: string, key: string) => void
+      fn(locale, key)
     }
```

Comment:
can you add a console.warn if we fall through without finding `handleMissingTranslation`?

Reply:
GC, done